### ISSUE TITLE
error when trying to use installer with 32 byte java could be more clear

### ISSUE THREAD

**Issue by jabo38** [NewEconomyMovement/NemCommunityClient#451]
![image](https://cloud.githubusercontent.com/assets/10395907/12287433/094e234c-ba10-11e5-9940-2819ef6a10f4.png)

Could this error, maybe tell people to check and make sure their java version is 64 bit instead of 32 bit.  Most people read this error as there is a wrong pathway or other problem.

**Comment by fyahfox** [comment]
I agree, it would be nice if the installer said: 32 bit detected, please install 64 bit.
